packages/discord.js/src/structures/Message.js

```diff
@@ -280,19 +286,19 @@ class Message extends Base {
        * Flags that are applied to the message
        * @type {Readonly<MessageFlags>}
        */
-      this.flags = new MessageFlags(data.flags).freeze();
+      this.flags = new MessageFlagsBitField(data.flags).freeze();
     } else {
-      this.flags = new MessageFlags(this.flags).freeze();
+      this.flags = new MessageFlagsBitField(this.flags).freeze();
     }
 
     /**
      * Reference data sent in a message that contains ids identifying the referenced message.
      * This can be present in the following types of message:
-     * * Crossposted messages (IS_CROSSPOST {@link MessageFlags.FLAGS message flag})
-     * * CHANNEL_FOLLOW_ADD
-     * * CHANNEL_PINNED_MESSAGE
-     * * REPLY
-     * * THREAD_STARTER_MESSAGE
+     * * Crossposted messages (IS_CROSSPOST {@link MessageFlags message flag})
+     * * `MessageType.ChannelFollowAdd`
+     * * `MessageType.ChannelPinnedMessage`
+     * * `MessageType.Reply`
+     * * `MessageType.ThreadStarterMessage`
```

Comment:
IMO this should keep the same pattern 
```suggestion
     * * `ChannelFollowAdd`
     * * `ChannelPinnedMessage`
     * * `Reply`
     * * `ThreadStarterMessage`
```

Reply:
I disagree, it should list values, and the enums are always prefixed with the enum type when used as a value.

Reply:
The typedef mentions "message types" right above, so you can deduce these are message types too. This is not meant to be a code representation of how one would use these types, it's simply a list of which ones a reference can be present in